content/participants/ttjc@Sakshamkapur/index.md

```diff
@@ -70,3 +68,15 @@ _stack_ built in html,css,javascript.
 _hosted link_  https://sakshamkapur.github.io/2048/2048.html
 
 _github link_ https://github.com/Sakshamkapur/2048
+
+#### Blogit: A Bloging Platform
```

Comment:
Hey! @Sakshamkapur Only projects done during the challenge are accepted for the challenge. As this was done 2 years ago please remove this project.